tensorflow/lite/kernels/add_n.cc

```diff
@@ -26,6 +26,32 @@ namespace add_n {
 constexpr int kInputTensor1 = 0;
 constexpr int kOutputTensor = 0;
 
+struct OpData {
+  int num_inputs;
+  void* all_inputs;
+  TfLiteType type;
```

Comment:
Suggestion: This can be a `template<typename T> struct OpData` that contains a single `VectorOfTensors<T>`. 

Alternatively, just return `VectorOfTensors<T>`

Reply:
@miaout17 : This is required, OpData can not be template based, as we dont know the datat type until prepare is executed. So we can not use in Init or Free, unless we know the datat type.